schema/site_metadata.json

```diff
@@ -50,6 +58,59 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
+            "description": "Area of the site in square meters",
+            "type": "number",
+            "examples": [1234, 256, 998765]
+        }
+      }
+    },
+    "source": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "folder": {
+          "description": "Folder where the UDMI site model is stored",
+          "type": "string",
+          "examples": ["/home/user/udmi_site_model", "https://drive.google.com/corp/drive/folders/iamadrivelocation"]
+        },
+        "repo": {
+          "description": "Source repository where the UDMI site model is stored",
+          "type": "string",
+          "examples": ["https://github.com/faucetsdn/udmi_site_model", "git@github.com:faucetsdn/udmi_site_model.git"]
+        }
+      }
+    },
+    "count": {
```

Comment:
"counts" (plural, since it's a collection of things) -- also -- do we want to scope this to "device_counts"? (not sure what other kinds of counts we might have).